Wrapping/Generators/Python/itk/support/itkExtras.py

```diff
@@ -272,8 +273,10 @@ def _GetImageFromArray(arr, function_name: str, is_vector: bool, ttype):
                     ImageType = itk.Image[itk.RGBPixel[itk.UC], Dimension]
                 elif VectorDimension == 4:
                     ImageType = itk.Image[itk.RGBAPixel[itk.UC], Dimension]
+                else:
+                    ImageType = itk.VectorImage[PixelType, Dimension]
             else:
-                ImageType = itk.Image[itk.Vector[PixelType, VectorDimension], Dimension]
+                ImageType = itk.VectorImage[PixelType, Dimension]
```

Comment:
Always returning a `VectorImage` even when `VectorDimension` is low (and thus we could continue to return a `Vector` as we have until now) ... is a pretty big change!

Reply:
Yes, this is a pretty big change in behavior, but VectorImage is required to support more channel types, and it is better to be consistent with the returned type instead of varying based on build options, number of channels. With 5.2, itk.VectorImage and itk.Image[itk.Vector is overall greatly improved and much more usable than it was before thanks to many of your changes -- it is better to make this change now with these improvements in usability.

Reply:
Perhaps we should use something conceptually along the lines of the following:  
```python3
try:
    ImageType = itk.Image[itk.Vector[PixelType, VectorDimension], Dimension]
    cast = ...
except:
    ImageType = itk.VectorImage[PixelType, Dimension]
    cast = ...
```
(The actual implementation would be a little more complicated given the current logic flow of the method being modified.)

Reply:
Never mind my last reply here.  Your answer works.